test/plugin_functional/plugins/core_history_block/kibana.json

```diff
@@ -0,0 +1,8 @@
+{
+  "id": "core_history_block",
+  "version": "0.0.1",
+  "kibanaVersion": "kibana",
+  "server": false,
+  "ui": true,
+  "requiredBundles": ["kibanaReact"]
+}
```

Comment:
Just to confirm my understanding of this - if we want to use `<Prompt>`, do we need to include this plugin in Enterprise Search as a required dependency? Or no/is it already baked in?

Reply:
I imported it from `react-router-dom` and it worked, so I guess it's baked in.

Reply:
Na, everything is provided from core. I just used `kibanaReact` in the test plugin for some utility functions